tests/unit/test_pathutil.py

```diff
@@ -44,7 +44,7 @@
     remove_dir_or_file
 )
 
-from tests.unit.conftest import MonkeyMock
```

Comment:
Anyone know why this import stopped working suddenly on GH Actions? The absolute import is still working locally for me

Reply:
It might be the difference between the editable install you will likely have for local development and the regular install on GH actions. I think the editable install will put the `tests` folder in `PYTHONPATH`.

The `tests` folder shouldn't be imported directly like that (hasn't been configured as a Python package, has no `__init__.py` so will be interpreted as a namespace package?).